src/java.desktop/share/classes/com/sun/media/sound/DLSSoundbank.java

```diff
@@ -31,6 +31,7 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.net.URL;
+import java.util.ArrayDeque;
```

Comment:
Can we remove the Stack from the list of imports?

Reply:
Removed